### ISSUE TITLE
Direct PouchDB Adapter

### ISSUE THREAD

**Issue by jo** [eHealthAfrica/kazana#192]
PouchDB Server is used for development and testing. Here we could save the cost of the http interface by directly providing the kazana config with a PouchDB adapter.

For example  `kazana.db.rawData` could support a full PouchDB adapter instead of the database name string:

```js
 config: {
   kazana: {
     db: {
       rawData: new PouchDB('http://localhost:5984/kazana-raw-data')
       // or
       // rawData: new PouchDB({ name: 'raw-data', adapter: 'memdown' })
     }
   }
}
```

**Comment by gr2m** [comment]
Yeah that was the initial plan but then run into several blockers. We are doing the same thing for the same reasons at Hoodie now, so we can join forces. One blocker was authentication, but we solved it now with https://github.com/hoodiehq/couchdb-calculate-session-id, https://github.com/hoodiehq/pouchdb-users and https://github.com/hoodiehq/pouchdb-admins. 

For usual databases, we are working on a generic reverse proxy solution which either proxies to a CouchDB url, or to a PouchDB instance. I’ve created an issue to track our progress here: https://github.com/hoodiehq/hoodie-server-store/issues/12
